spec/features/user_registers_new_application_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature "User registers new application", js: true do
+  scenario "successfully" do
+    user = create(:user)
+
+    visit new_user_session_path
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+
+    click_button "Log in"
+
```

Comment:
The login steps for lines 7 to 11 are repeated in a few places.

Consider moving the login code to a dedicated helper method in a `LoginHelper` in `spec/support/login_helper.rb`.

`LoginHelper` can follow a similar pattern that you worked with in https://github.com/jeffminnear/waw-api/blob/master/spec/support/json_helper.rb